sdk/keyvault/azsecrets/testdata/recordings/TestBeginDeleteSecretRehydrated.json

```diff
@@ -0,0 +1,732 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://fakekvurl.vault.azure.net/secrets/begindeleterehydrat3511093089?api-version=7.3",
+      "RequestMethod": "PUT",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip",
+        "Content-Length": "0",
+        "User-Agent": "azsdk-go-internal/v0.7.0 azsdk-go-azcore/v0.23.0 (go1.18; Windows_NT)"
```

Comment:
Shouldn't the test fail now that the client's User-Agent doesn't match the recording?

Reply:
By default we don't check the User-Agent header because then we'd have to regen for every version update to core and each service.